### ISSUE TITLE
re implementer le ping windows

### ISSUE THREAD

**Issue by rodesousa** [rodesousa/luz-lantern#22]

**Comment by ptavares** [comment]
Après analyse code, il suffit de mettre une fonction en paramètre du Type Shard, qui permettra l'analyse du retour de la commande executer car pour le ping windows, il ne retourne pas l'erreur.
Cela permettrai en plus gerer les cas particulier

**Comment by rodesousa** [comment]
un echo true ?

**Comment by ptavares** [comment]
non une fonction dans le type shard genre 

```
type Shard struct {
    Name     string
    Cmd_line []string
    Args     ShardArguments
    Status   Result
    Analyse func(arg string, out string, err error) (bool, error)
}
```

Avec une generique (en gros ce qui est fait dans le traitement de la fonction cmd() ) et une specifique pour le windows dans le cas particulier du nslookup.

Ca à l'air assez simple à faire, je suis en cours de prototypage la.

**Comment by ptavares** [comment]
En plus, de cette facon, il est sera simple de customiser un peu le message d'erreur pour le récapitulatif et donc resoudre l'issue #24 par la même occasion non ?

**Comment by rodesousa** [comment]
yeah pour #24. Par contre avant de faire cela et la #24 il faut review la #25
